examples/notebooks/notebook_test_scripts/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
Should we be moving these instead of just adding them in the new location? Could you `git rm` the old scripts?

Reply:
ah sorry - I didn't push that change. The new commit should reflect it.